docs/document/docs/components/desktop/table.md

```diff
@@ -6,5 +6,133 @@ original_id: table
+| title | string | ""  | 	Header name of the column | |
+| requiredIcon | boolean | false  | Show/Hide the required icon | |
+| visible | boolean |  true  | Show/Hide the column | |
+| render | function <br> `function(cellData, rowData, rowIndex) {}` | - | Renderer of the cell | <ul><li>The return value should be a DOM</li><li>`cellData` is the data of the current cell rendered</li><li>`rowData` is the data of the current row rendered</li><li>`rowIndex` is the index of the current row rendered</li></ul> 3 params above provide more information for the function.</br>If `render` function is not specified, the cell will display with the default text.|
+
+
+### Event
+
+Here is a list of events that can be specified:
+
+| Name | Type | Description | Remark |
+| :--- | :--- | :--- | :--- |
+| change | function | The function fire when the table change. | `function(event: Event => void)` <br><br> - It will pass the event object as the argument<br> - You can receive the following values in event.detail <ul><li>`change-cell` (Triggered if the data in cell is changed)</li><ul><li>event.detail.`type` : change-cell</li><li>event.detail.`rowIndex`: the row's index has changed</li><li>event.detail.`data`: all data of table</li><li>event.detail.`oldData`: old data of table</li><li>event.detail.`field`: The columns has changed</li></ul><li>`add-row` (Triggered if button add row is clicked)</li><ul><li>event.detail.`type` : add-row</li><li>event.detail.`rowIndex`: The row’s index added.</li><li>event.detail.`data`: all data of table</li><li>event.detail.`oldData`: old data of table</li></ul><li>`remove-row` (Triggered if button remove row is clicked</li><ul><li>event.detail.`type` : remove-row</li><li>event.detail.`rowIndex`: The row’s index removed.</li><li>event.detail.`data`: all data of table</li><li>event.detail.`oldData`: old data of table</li></ul></ul>|
+
+### Constructor
+
+Table(options)<br>
+Here is a list of available constructors:
+
+#### Parameter
+
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| options | object | {} | Object that includes component properties | |
+
+---
+
+## Sample Code
+
+> Please check the [package installation](../../getting-started/quick-start.md#installation) method first.
+
+Here is a sample code when all parameters are specified.
+
+```javascript
+const Kuc = Kucs['1.x.x'];
+
+const header = kintone.app.getHeaderMenuSpaceElement();
+
+const renderAge = (dataCell) => {
+  const spanElement = document.createElement("span");
+  spanElement.innerText = `The age is ${dataCell}`;
+  return spanElement;
+};
+
+const renderName = (cellData) => {
+  const dropdown = new Dropdown({
+    items: [
+      { label: 'John Brown', value: 'John Brown' },
+      { label: 'Steven Gerrard', value: 'Steven Gerrard' },
+    ],
+    value: cellData,
+    selectedIndex: 0
```

Comment:
Do we need this prop?

Reply:
We don't need that prop, I will remove it.

Reply:
Confirmed